src/utils/makeReactNativeConfig.js

```diff
@@ -197,6 +213,10 @@ const getDefaultConfig = (
     mainFields: ['react-native', 'browser', 'main'],
     extensions: [`.${platform}.js`, '.native.js', '.js'],
   },
+  /**
+   * Set target to webworker as it's closer to RN environment than `web`.
+   */
+  target: 'webworker',
```

Comment:
> it uses different mechanism for downloading files

nice

> there are other differences in other files

what are the other differences? do they impact anything else than script loading mechanism?